applications/asset_tracker_v2/src/modules/Kconfig.debug_module

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig DEBUG_MODULE
+	bool "Debug module"
+
+if DEBUG_MODULE
+
+config DEBUG_MODULE_USE_CUSTOM_TRANSPORT
+	bool "Use custom transport when sending memfault data"
+	help
+	  If this option is disabled, Memfault firwmare SDKs own internal HTTP transport is used to
```

Comment:
```suggestion
         If enabled, the debug module will forward the data in chunks via
	  events of type DEBUG_EVT_MEMFAULT_DATA_READY via the
	  configured cloud to Memfault. If disabled, the debug module will use the 
	  Memfault firmware SDKs internal HTTP transport. This can be used to save overhead 
	  associated with maintaining multiple connections.```